gp-includes/things/translation.php

```diff
@@ -576,7 +576,7 @@ function( $s ) use ( $all_statuses ) {
 			for ( $i = 0; $i < $locale->nplurals; $i++ ) {
 				$row->translations[] = $row->{'translation_' . $i};
 			}
-			$row->references         = preg_split( '/\s+/', $row->references, -1, PREG_SPLIT_NO_EMPTY );
+			$row->references         = null !== $row->references ? preg_split( '/\s+/', $row->references, -1, PREG_SPLIT_NO_EMPTY ) : array();
```

Comment:
These would be alternatives:

```suggestion
			$row->references         = $row->references ? preg_split( '/\s+/', $row->references, -1, PREG_SPLIT_NO_EMPTY ) : array();
```

```suggestion

			if ( $row->references ) {
				$row->references = preg_split( '/\s+/', $row->references, -1, PREG_SPLIT_NO_EMPTY );
			} else {
				$row->references = array();
			}	
```